pkg/kubelet/kubelet_linux.go

```diff
@@ -49,3 +56,52 @@ func (kl *Kubelet) cgroupVersionCheck() error {
 	}
 	return nil
 }
+
+func (kl *Kubelet) startHealthCheck() error {
+	if runtime.GOOS != "linux" {
+		return fmt.Errorf("systemd watchdog can only be used on linux")
+	}
+
+	watchdogVal, err := daemon.SdWatchdogEnabled(false)
+	if err != nil {
+		return fmt.Errorf("could not check if systemd watchdog information: %v", err)
```

Comment:
should the kubelet fail to start in this case? If kubelet will start, and watchdog is enabled, systemd may keep killing the kubelet